### ISSUE TITLE
Make ME respect write protection

### ISSUE THREAD

**Issue by psonnosp** [Komet/MediaElch#948]
**Is your feature request related to a problem? Please describe.**
On some shows I create my own actors images and on long running show you need to update the list of actors from time to time. Problem is that ME doesn't care if a file is write protected and gladly delete the write protected files and replace them with the once that are scraped instead of just ignoring them and adding new actors if there are any.

**Describe the solution you'd like**
That ME respect if an image is write protected and doesn't overwrite them.

**Describe alternatives you've considered**
I usually have a backup to avoid this but sometime you forget about it and then it's to late...

**Additional context**
If this make something else go bananas maybe a scrape button on the actors page with a "Just add new" check box would be a possibility.

**Comment by bugwelle** [comment]
Wow. I'm surprised that the write operation does not fail if the file is protected. Just to be sure: How did you add write-protection to the file?

If MediaElch overwrites a protected file, is is not only strange but likely a bug.

**Comment by bugwelle** [comment]
Ok, so I tested it on my Windows 8 machine.  
However, if I write-protect a single file, MediaElch does not overwrite it. :-/

Here the steps I took (I hope I've got the English names correct, my Windows system is German)

1. Go to movie directory
1. Right click on poster -> Properties -> Attributes -> Write Protection: Enable
1. MediaElch -> movie -> select new poster or use the "Screenshot" button
1. Click "Save"
1. Movie poster reverts to old one
